generated/nidcpower/nidcpower_service.cpp

```diff
@@ -11,6 +11,7 @@
 #include <iostream>
 #include <atomic>
 #include <vector>
+#include <server/converters.h>
```

Comment:
I'm including my helper unconditionally even though only a few services use it now. I'd like it to be easier to add standard utilities there since I think it's easier to grok than generating them inline.